modules/ROOT/pages/monitoring/metrics/reference.adoc

```diff
@@ -437,6 +437,17 @@ label:deprecated[Deprecated in 5.15]
 |<prefix>.cluster.discovery.restart.failed_count|Discovery restart failed count. (gauge)
 |===
 
+[[lighthouse-metrics]]
+=== Discovery metrics v2
+
+.Discovery metrics v2
```

Comment:
```suggestion
.Lighthouse metrics
```

Reply:
> The title should reflect the code `@Documented(".Lighthouse metrics")`.

https://github.com/neo4j/docs-operations/pull/1692#issuecomment-2196629424

>What about the rest of the metrics? Will we document them as well for 5.22 or just these two?

Discovery service V2(Lighthouse) has only two public metrics. All other metrics that you see in the file are internal. We don't document internal metrics.